.github/workflows/security-analysis.yml

```
@@ -1,14 +1,41 @@
+ feat/integrate-alibaba-access-analyzer-12183567303830527494
+name: Security Analysis
```

actionlint (1.7.10)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)